config/config.yml

```diff
@@ -27,10 +27,12 @@ vagrant_synced_folders:
     destination: "{{ drupal_composer_install_dir }}"
     type: nfs
     create: true
+    mount_options: ['nolock', 'vers=3', 'tcp', 'fsc', 'rw', 'noatime']
   - local_path: "{{ drupal_composer_site_host_dir }}"
     destination: "{{ drupal_composer_site_install_dir }}"
     type: nfs
     create: true
+    mount_options: ['nolock', 'vers=3', 'tcp', 'fsc', 'rw', 'noatime']
```

Comment:
Can you remove this dupe line please?

Reply:
That's not a duplicate line - there's two drives being mounted with NFS, so need to include the config in twice.